src/app/app.component.html

```diff
@@ -1,18 +1,14 @@
 <div class="root" fxLayout="column">
   <mat-toolbar color="primary">
     <span>
-      <img class="logo" alt="Ginkgo" src="assets/logo.png">
-      <span>
-      Performance Explorer
-      </span>
+      <img class="logo" alt="Ginkgo" src="assets/logo.png" />
+      <span> Performance Explorer </span>
     </span>
     <span class="fill-space"></span>
-    <a mat-icon-button href="https://github.com/ginkgo-project/gpe/wiki"
-       target="_blank">
+    <a mat-icon-button href="https://github.com/ginkgo-project/gpe/wiki" target="_blank">
```

Comment:
Is the wiki up to date ?

Reply:
There seems to be formatting issues, but it still is. Nothing really changed in terms of interface for the GPE.